tools/data/super-resolution/div2k/preprocess_div2k_dataset.py

```diff
@@ -31,7 +32,7 @@ def main_extract_subimages(args):
 
     Usage:
         For each folder, run this script.
-        Typically, there are four folders to be processed for DIV2K dataset.
+        Typically, there are four folders to be processed for DIV2K dataset according to scale factor list ([2,3,4])
```

Comment:
By default, there are four folders to be processed for DIV2K dataset according to scale factor list ([2,3,4])
  DIV2K_train_HR
  DIV2K_train_LR_bicubic/X2
  DIV2K_train_LR_bicubic/X3
  DIV2K_train_LR_bicubic/X4
After processing, each sub_folder should have the same number of subimages.
You can also process a subset of folders by modifying the argument `scale`. Remember to modify opt configurations according to your settings.

(May need to format it)